Cargo.toml

```diff
@@ -260,6 +262,8 @@ test = false
 doc = false
 
 [features]
+default = ["mimalloc"]
```

Comment:
Custom allocator should not be default. Opt-in is a better option to respect people's choices.